import "Event.proto";

package com.lvl6.proto;

option java_outer_classname = "ProtocolsProto";

enum EventProtocolRequest {
	C_CHAT_EVENT = 0;
	C_BATTLE_EVENT = 1;
	C_VAULT_EVENT = 2;
	C_TASK_ACTION_EVENT = 3;
	C_CLERIC_HEAL_EVENT = 4;
	C_STARTUP_EVENT = 5;
	C_RETRIEVE_TASKS_FOR_CITY_EVENT = 6;
	C_RETRIEVE_QUESTS_FOR_CITY_EVENT = 7;
	C_RETRIEVE_EQUIPS_FOR_ARMORY_EVENT = 8;
	C_ARMORY_EVENT = 9;
	C_IN_APP_PURCHASE_EVENT = 10;
	C_RETRIEVE_CURRENT_MARKETPLACE_POSTS_EVENT = 11;
	C_POST_TO_MARKETPLACE_EVENT = 12;
	C_RETRACT_POST_FROM_MARKETPLACE_EVENT = 13;	
	C_PURCHASE_FROM_MARKETPLACE_EVENT = 14;	
	C_USE_SKILL_POINT_EVENT = 15;
	C_GENERATE_ATTACK_LIST_EVENT = 16;
}

enum EventProtocolResponse {
	S_CHAT_EVENT = 0;
	S_BATTLE_EVENT = 1;
	S_VAULT_EVENT = 2;
	S_TASK_ACTION_EVENT = 3;
	S_CLERIC_HEAL_EVENT = 4;
	S_STARTUP_EVENT = 5;
	S_RETRIEVE_TASKS_FOR_CITY_EVENT = 6;
	S_RETRIEVE_QUESTS_FOR_CITY_EVENT = 7;
	S_RETRIEVE_EQUIPS_FOR_ARMORY_EVENT = 8;
	S_ARMORY_EVENT = 9;
	S_IN_APP_PURCHASE_EVENT = 10;
	S_RETRIEVE_CURRENT_MARKETPLACE_POSTS_EVENT = 11;
	S_POST_TO_MARKETPLACE_EVENT = 12;
	S_RETRACT_POST_FROM_MARKETPLACE_EVENT = 13;	
	S_PURCHASE_FROM_MARKETPLACE_EVENT = 14;	
	S_USE_SKILL_POINT_EVENT = 15;
	S_GENERATE_ATTACK_LIST_EVENT = 16;	
	
	S_LEVEL_UP_EVENT = 19;
	S_UPDATE_CLIENT_USER_EVENT = 20;	
}